docs/sources/alert/_index.md

```diff
@@ -106,6 +106,40 @@ This query (`expr`) will be executed every 1 minute (`interval`), the result of
 name we have defined (`record`). This metric named `nginx:requests:rate1m` can now be sent to Prometheus, where it will be stored
 just like any other metric.
 
+
+### Limiting Alerts AND Series
+
+Like Prometheus, A limit for alerts produced by alerting rules and series produced by recording rules can be configured per-group in Loki also. When the limit is exceeded, all series produced by the rule are discarded, and if it's an alerting rule, all alerts for the rule, active, pending, or inactive, are cleared as well. Default value for limit is **0** i.e. no limit.
```

Comment:
> Like Prometheus

Please link to Prometheus' documentation.

Also, I think we should describe _why_ someone should use this as well as simply _how_.

Reply:
tried to explain a bit more here. please check.